server/server.go

```diff
@@ -499,8 +502,15 @@ git commit plugins.json data/statik/statik.go -m "Add %[1]s of %[2]s to the Mark
 git push --set-upstream origin %[3]s
 git checkout master
 `, tag, repo, branch)
-		url := fmt.Sprintf("https://github.com/mattermost/mattermost-marketplace/compare/production...%s?quick_pull=1&labels=3:+QA+Review,2:+Dev+Review", branch)
-		msg = fmt.Sprintf("Plugin was successfully signed and uploaded to Github and S3.\nTag: **%s**\nRepo: **%s**\n[Release Link](%s)\nTo add this release to the Plugin Marketplace run inside your local Marketplace repository:\n```%s\n```\nUse %s to open a Pull Request.", tag, repo, releaseURL, marketplaceCommand, url)
+
+		url := fmt.Sprintf(
+			"https://github.com/mattermost/mattermost-marketplace/compare/production...%s?quick_pull=1&labels=3:+QA+Review,2:+Dev+Review",
+			branch,
+		)
+		msg = fmt.Sprintf(
+			"Username: `@%s`\nPlugin was successfully signed and uploaded to Github and S3.\nTag: **%s**\nRepo: **%s**\n[Release Link](%s)\nTo add this release to the Plugin Marketplace run inside your local Marketplace repository:\n```%s\n```\nUse %s to open a Pull Request.",
```

Comment:
What values does repeating the username here provide?

Reply:
Just so everything is in one spot, the user doesn't have to look for. the initial post to see who initiated the build.

And also https://github.com/mattermost/matterbuild/pull/45#discussion_r457697905

Reply:
@hanzei @ali-farooq0 Also, if I'm not mistaken, another post could get in between the previous post with the username and this one since this call is being made asynchronously from a separate goroutine.

Reply:
Correcto. Which is why I recommend adding it here aswell